src/services/gererateErrorResponse.ts

```diff
@@ -0,0 +1,15 @@
+import { ErrorResponse } from '../lambdas/types'
+
+export const generateErrorResponse = (error: unknown): ErrorResponse => {
+  const response = {
+    statusCode: 500,
+    message: 'Something went wrong',
+  }
+  if (error instanceof Error) {
+    if (error.name === 'RequestTimeOut') {
+      response.statusCode = 408
+      response.message = 'Request to externial API timed out'
+    }
+  }
+  return response
+}
```

Comment:
This is excellent error handling 👍 

Two things worth mentioning though:
- We can just return from within the if statement so we don't need to reassign values on the constant
- The error name would change if this were an actual abort error, I think once the code from my other feedback is in place this would need to change 🙂

Reply:
I updated the condition looking at error.name to look for AbortError instead of my made-up magic name 😬 . I removed the response const and just have objects being directly returned in the if statement and as a default.